components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/index.tsx

```diff
@@ -71,25 +50,24 @@ function ConversationEntries(props: ConversationEntriesProps) {
     onTouchMove: () => setActiveMenuId(null)
   })
 
-  React.useEffect(() => {
-    if (lastEntryElementRef.current === null) return
-    if (!conversationContext.isGenerating) return
-    props.onLastElementHeightChange()
-  }, [conversationContext.conversationHistory.length, lastEntryElementRef.current?.clientHeight])
-
   const lastAssistantId = React.useMemo(() => {
     // Get the last entry that is an assistant entry
-    for (let i = conversationContext.conversationHistory.length - 1; i >= 0; i--) {
+    for (
+      let i = conversationContext.conversationHistory.length - 1;
+      i >= 0;
+      i--
+    ) {
       if (
-        conversationContext.conversationHistory[i].characterType === mojom.CharacterType.ASSISTANT
+        conversationContext.conversationHistory[i].characterType ===
+        Mojom.CharacterType.ASSISTANT
```

Comment:
nit: indent on newline
```suggestion
          Mojom.CharacterType.ASSISTANT
```